akka-persistence-typed/src/test/resources/logback-test.xml

```diff
@@ -5,7 +5,7 @@
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <pattern>%date{ISO8601} %-5level %logger %marker - %msg MDC: {%mdc}%n</pattern>
+            <pattern>%date{ISO8601} %-5level %logger %marker %X{akkaSource} - %msg %n</pattern>
```

Comment:
add the persistence specific mdc values explicity?

Reply:
👍 

I added persisetnceId left out phase for now